src/controller/investments.js

```diff
@@ -1,15 +1,19 @@
 const express = require('express');
-const userInvestments = require('../models/data');
+const jwtVerifier = require('express-jwt');
+
+const secret = process.env.ACCESS_TOKEN_SECRET;
 
 const investments = express.Router();
+const { User } = require('../models');
 
-investments.get('/user/:uid', (req, res) => {
-  const { uid } = req.params;
-  const userData = userInvestments.find((data) => data.userId === uid);
-  if (userData) {
-    return res.send(userData);
+investments.get('/user', jwtVerifier({ secret }), async (req, res) => {
+  try {
+    const { _id } = req.user;
+    const user = await User.findById(_id, 'investments');
```

Comment:
are you sure we don't need to transfer the data? Don't we have an `_id` in the user document? All the keys are in `camelCase` and has only the mandatory fields? Aren't there any mongo related fields in the document?